e2e/tauri/specs/auth.e2e.js

```diff
@@ -0,0 +1,61 @@
+const fs = require('fs/promises')
+
+describe('KCMA (Tauri, Linux)', () => {
+  it('opens the auth page, signs in, and signs out', async () => {
+    // Clean up previous tests
+    await new Promise((resolve) => setTimeout(resolve, 100))
+    await fs.rm('/tmp/kittycad_user_code', { force: true })
+    await browser.execute('window.localStorage.clear()')
+
+    const signInButton = await $('[data-testid="sign-in-button"]')
+    expect(await signInButton.getText()).toEqual('Sign in')
+
+    // Workaround for .click(), see https://github.com/tauri-apps/tauri/issues/6541
+    await signInButton.waitForClickable()
+    await browser.execute('arguments[0].click();', signInButton)
+    await new Promise((resolve) => setTimeout(resolve, 2000))
+
+    // Get from main.rs
+    const userCode = await (
+      await fs.readFile('/tmp/kittycad_user_code')
+    ).toString()
+    console.log(`Found user code ${userCode}`)
+
+    // Device flow: verify
+    const token = process.env.KITTYCAD_API_TOKEN
+    const headers = {
+      Authorization: `Bearer ${token}`,
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    }
+    const verifyUrl = `https://api.kittycad.io/oauth2/device/verify?user_code=${userCode}`
+    console.log(`GET ${verifyUrl}`)
+    const vr = await fetch(verifyUrl, { headers })
```

Comment:
Why does the token need to be included in the headers when hitting the `oauth2/device/verify` endpoint? just seems like it shouldn't be needed when testing the auth flow.

Reply:
Because those requests need to be authenticated I believe! When the URL pops open in a browser, you need to sign in to kittycad first and then approve (aka verify + confirm)

Reply:
Oh right that makes sense.